deployment/tf/modules/v2/worker-nodegroup.tf

```diff
@@ -81,6 +81,51 @@ resource "yandex_kubernetes_node_group" "workers-s" {
   }
 }
 
+resource "yandex_kubernetes_node_group" "workers-l" {
+  cluster_id  = yandex_kubernetes_cluster.allocator_cluster.id
+  name        = "workers-l"
+  description = "Nodegroup for lzy workers with label l"
+  node_labels = {
+    "lzy.ai/node-pool-id" = "l1"
+    "lzy.ai/node-pool-label" = "l"
```

Comment:
let's use upper case letter for names

Reply:
I will change it in other PR to not reapply this terraform to dev installation